covalent/executor/base.py

```diff
@@ -147,6 +149,16 @@ def __init__(
         self.time_limit = time_limit
         self.retries = retries
 
+        if not workdir:
+            try:
+                workdir = get_config(f"executors.{self.short_name()}.workdir")
+            except KeyError:
+                workdir = os.path.join(os.environ["HOME"], "covalent", "workdir")
+                info_msg = f"Couldn't find `executors.{self.short_name()}.workdir` in config, will create a default one at {workdir}"
+                app_log.debug(info_msg)
```

Comment:
I have used it, let me know if it should be removed...

Reply:
@madhur-tandon As long as it's defined in the `_EXECUTOR_PLUGIN_DEFAULTS` the `executor_manager` in `covalent/executor/__init__.py` will take care of populating the config file with those defaults. So I would say you don't even have to check whether the key exists or not. Because if it isn't there that means the problem lies somewhere else as things aren't working as they're supposed to 😅 .